app/src/main/java/ro/code4/deurgenta/helper/Utils.kt

```diff
@@ -129,6 +132,14 @@ fun hideSoftInput(view: View) {
     imm.hideSoftInputFromWindow(view.windowToken, 0)
 }
 
+fun takeUserTo(url: String, currentActivity: Activity) {
```

Comment:
Considering the other functions in this module, would it be better to turn this into an `Activity.takeUserTo` extension method?